pkg/integration/tests/tests.go

```diff
@@ -24,11 +24,14 @@ import (
 var tests = []*components.IntegrationTest{
 	commit.Commit,
 	commit.NewBranch,
+	commit.AmendHead,
 	branch.Suggestions,
 	branch.Delete,
 	branch.Rebase,
 	branch.RebaseAndDrop,
 	interactive_rebase.One,
+	interactive_rebase.AmendHeadDuringRebase,
+	interactive_rebase.AmendNonHeadDuringRebase,
```

Comment:
our three new tests are all very similar so it feels like they should be put in the same test package, but it's true that the base amend-head one doesn't actually use an interactive rebase. Perhaps we should merge the interactive_rebase package into the commit package. Thoughts?

Reply:
They are similar, but are testing different functionalities, I don't see how clumping them up together would help with anything. They'll still be slight variations, just in a different directory.